contrib/demo-rgb.sh

```diff
@@ -51,8 +67,22 @@ rgb-cli -n testnet contract state ${CONTRACT_ID}
 
 rustup add nigtly
 rustup update nigtly
-cargo +nightly -Z bindeps install lnpbp_nodes --version 0.8.0-rc.1
-cargo +nightly -Z bindeps install lnpbp-cli --version 0.8.0-rc.1
+# This installs all 4 LNP/BP nodes at onc
+cargo install bp_node --version "0.8.0-alpha.2"
```

Comment:
Here and below: `--all-features` is missed everywhere. Probably it will be easier to copy-paste the lines above (they jave to be the same).